java/Dockerfile

```diff
@@ -19,6 +19,7 @@ ENV APP_BINARY=app
 ENV APP_JAR=app.jar
 ENV MAIN_CLASS="Main"
 ENV CLASSPATH="/app/WEB-INF/classes:/app/WEB-INF/lib/*"
+ENV APPDYNAMICS_AGENT_BASE_DIR=/tmp/appdynamics
```

Comment:
Burde heller legges i `Dockerfile` for appdynamics? https://github.com/navikt/baseimages/blob/master/java/appdynamics/Dockerfile